package.json

```diff
@@ -80,11 +88,12 @@
     "rollup-plugin-terser": "~7.0.2",
     "serve": "~12.0.0",
     "sinon": "~11.1.1",
-    "sinon-chai": "~3.7.0",
-    "watchify": "~4.0.0"
+    "sinon-chai": "~3.7.0"
   },
   "dependencies": {
+    "@size-limit/preset-small-lib": "^5.0.1",
```

Comment:
These should be devDependencies. But in any case, I don't particularly want to add these.

Reply:
Outdated change, I already fixed it, but I can remove it.